weave-js/src/components/Panel2/PanelDateRange.tsx

```diff
@@ -29,25 +29,31 @@ const inputType = {
     },
   ],
 };
-
+const StyledConfigOpt = styled(ConfigPanel.ConfigOption)`
+  && {
+    font-size: 15px;
+  }
+`;
```

Comment:
For any new styled components code I'd like to recommend we set the display name like:
`StyledConfigOpt.displayName = 'S.StyledConfigOpt';`
I find this really useful when debugging and looking at things in the react component tree.

Reply:
oh yes my b will add some now